ios/DdLogs.m

```diff
@@ -4,7 +4,7 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
-@import React;
+#import <React/RCTBridgeModule.h>
```

Comment:
does `@import React;` give errors? 
this change was a fix attempt when `RCTBridgeModule` caused compilation errors in Alex's local
i had the same problem once in my local but i couldn't reproduce it consistently

Reply:
This class is automatically generated from `dd-mobile-bridge` tasks. If this **needs** to be changed, we should change it there and run the generation code again. 

That being said: 
- I didn't get any issue when running my tests in iOS. 
- This is the official recommended import when building native modules